template/dispatch.c.erb

```diff
@@ -17,16 +18,17 @@ btx_dispatch_<%= e.name_sanitized %>(UT_array *callbacks,
   // Their type are declared in 'dispatch.h'
   <%= e.name_sanitized %>_callback_f **p = NULL;
   while ((p = utarray_next(callbacks, p))) {
-    (*p)(common_data, common_data->usr_data,
+    (*p)((void *)common_data, common_data->usr_data,
```

Comment:
I aggree the idea of put more meaningfull names, but if that may add more complexity to the code, I am not sure if it worth it, even when we have to cast everywhere, what do you think? or maybe I am not seeing the big picture.

Reply:
It's more about "separation of concern." The user of metababel, should not need to care about the internal that we use inside metababel.